packages/acs-ui-common/src/identifier.ts

```diff
@@ -8,6 +8,13 @@ import {
   isPhoneNumberIdentifier
 } from '@azure/communication-common';
 
+const COMMUNICATION_USER_PREFIX = '8:acs:';
+const PHONE_NUMBER_PREFIX = '4:';
+const TEAMS_DOD_PREFIX = '8:dod';
```

Comment:
These should all include trailing `:`. Add tests that would fail, then fix this.